cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
+describe('Article', () => {
+  let user;
+  let articleTest;
+
   before(() => {
```

Comment:
It would be a good practice to move the generateUser and generateArticle tasks to a beforeEach hook to ensure unique data for each test case.